mods/beds/api.lua

```diff
@@ -59,8 +59,8 @@ function beds.register_bed(name, def)
 				return itemstack
 			end
 
-			local def = minetest.registered_nodes[minetest.get_node(pos).name]
```

Comment:
Well, the upvalue `def` was shadowed by the local variable. That is often no problem, until someone tries to access the upvalue and receives the local variable instead. The linter reports on these issues. Redefinitions of variables within the same function are reported differently and were fixed by the other commit before, mostly by reassigning instead of redefining.